tools/packaging/kernel/configs/fragments/common/fs.conf

```diff
@@ -52,8 +52,6 @@ CONFIG_AIO=y
 # Docker in Docker support requires overlay
```

Comment:
@skaegi, just reading the comment here makes me wonder ... does the change break docker in docker support?

Reply:
no -- both ways work with docker-in-docker
The difference is that when you do a `docker info` for the Storage driver.
```
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Using metacopy: false
  Native Overlay Diff: true
  userxattr: false
```  
you'll either see `Native Overlay Diff` or `Using metacopy` set to `true` but not both. Native Overlay Diff is "much" more valuable for performance in typical use.

Reply:
ah ok -- rebuilt the kernel and saw Native Overlay Diff again disabled. More info here -- https://github.com/torvalds/linux/commit/d47748e5ae5af6572e520cc9767bbe70c22ea498#diff-315c61b86e39b9b47f4ab0cd9efc2467

So... both need to be removed to enable Native Overlay Diff.